### ISSUE TITLE
Create an IntelliJ/AndroidStudio plugin for the converter

### ISSUE THREAD

**Issue by angelolloqui** [angelolloqui/SwiftKotlin#74]
When pasting Java code to a Kotlin file IntelliJ will automatically translate the code into Kotlin.

It would be ideal if something similar can be created for Swift to Kotlin via Plugin. However, so far I have seen that they only accept JVM based plugins, so SwiftKotlin can not be embedded. An idea (not explored yet) is to run the tool as command line external execution, but it will introduce restrictions as it can only be run under certain operating systems (Mac and Linux), and it will require the tool installation outside the plugin somehow. 

Research if this "solution" is workable or if there are any others available that adapts better to this case.

https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started.html

**Comment by romakononovich** [comment]
+++

**Comment by beilly** [comment]
Is there any linux command tools?

**Comment by angelolloqui** [comment]
> Is there any linux command tools?

AFAIK SwiftKotlin command line tool has not been tested in Linux, and my guess is that it will not work out of box. However, dependencies used are Linux compatible (I believe) and therefore it should be possible to make it run on Linux with not much work. How does that relate to this issue?

**Comment by githubashutoshsoni** [comment]
yes running this on ubuntu would be great. idk haven't tested it yet but gonna give it a go.
